libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp

```diff
@@ -99,7 +104,32 @@ class rc_delegation_pool_object : public object< rc_delegation_pool_object_type,
       id_type                       id;
 
       account_name_type             account;
+      asset_symbol_type             asset_symbol;
       steem::chain::util::manabar   rc_pool_manabar;
+      int64_t                       max_rc = 0;
+};
+
+/**
+ * Represents the total amount of an asset delegated by a user.
```

Comment:
Are we allowing other types of staked assets to contribute to RCs? I know we planned to allow users to delegate to SMTs to fund emissions and other automated actions.